odoo/addons/base/models/ir_rule.py

```diff
@@ -192,20 +192,27 @@ def _make_access_error(self, operation, records):
 
         model = records._name
         description = self.env['ir.model']._get(model).name or model
-        msg_heads = {
-            # Messages are declared in extenso so they are properly exported in translation terms
-            'read':   _("Due to security restrictions, you are not allowed to access '%(document_kind)s' (%(document_model)s) records.", document_kind=description, document_model=model),
-            'write':  _("Due to security restrictions, you are not allowed to modify '%(document_kind)s' (%(document_model)s) records.", document_kind=description, document_model=model),
-            'create': _("Due to security restrictions, you are not allowed to create '%(document_kind)s' (%(document_model)s) records.", document_kind=description, document_model=model),
-            'unlink': _("Due to security restrictions, you are not allowed to delete '%(document_kind)s' (%(document_model)s) records.", document_kind=description, document_model=model)
+        operation_error = _("Uh-oh! Looks like you have stumled upon some top-secret %ss.", description)
+        resolution_info_per_operation = {
+            'read':   _("If you really, really need access, go bribe your friendly administrator"),
+            'write':  _("But don’t worry, a little birdie told me that if you try to charm the administrators with some humor, they might just grant you that coveted access you seek!"),
```

Comment:
here we have lost information: this message doesn't contain the fact that you can't write